reactioncommerce/reaction
# (Feat) Create deploy to heroku button

Type: feature

## Issue
This adds a 'Deploy to Heroku' button in our readme.

## Solution
Created `app.json` as required in https://devcenter.heroku.com/articles/heroku-button, then modified our readme.

**Do not merge until release of v1.13.0**
Lazy loading of `transliteration` package in v1.12.1 causes the app build to fail, but with the changes in v1.13.0, the build goes through successfully.

## Breaking changes
None

## Testing
1. Click 'Deploy to Heroku' button in readme.
2. Fill out information required by Heroku.
3. Wait for some time (roughly 30 mins), the app should successfully build.

(Until 1.13.0 is released, test by creating a fork and manually adding app.json and updating the readme into `release-1.13.0` branch.)
